lib/containers/container_images.pm

```diff
@@ -63,6 +63,12 @@ sub build_with_zypper_docker {
     die 'Argument $image not provided!'   unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
 
+    # The zypper-docker works only on openSUSE or on SLE based image on SLE host
+    unless ((check_os_release('SUSE Linux Enterprise', 'PRETTY_NAME') && check_os_release('SUSE Linux Enterprise', 'PRETTY_NAME', runtime => $runtime, target => $image)) || check_os_release('openSUSE', 'PRETTY_NAME', runtime => $runtime, target => $image)) {
```

Comment:
Actually another way how to check the host OS `rpm -q kernel-default --qf "%{Distribution}\n"`. Just a quick idea, maybe it is worthless. :)

Reply:
The problem is we now test SLE based images on Ubuntu too 😄 
But I definitely agree this line is too long and should simplified.

Reply:
Quick search yields `dpkg-deb -I` but haven't gone deeper. Just brainstorming :)

Reply:
What are disadvantages of `/etc/os-release` in your opinion?

Reply:
i know that i agreed to have this enhancement on `check_os_release` because i think it can be handy but clearly you can just use `can_build_sle_base` here. Another proposal/idea would be to modify `can_build_sle_base` and wrap there the long conditions, removing the previous implementetion. WDYT?

Reply:
> /etc/os-release

https://www.freedesktop.org/software/systemd/man/os-release.html

```
Operating system vendors may extend the file format and introduce new fields. It is highly recommended to prefix new fields with an OS specific name in order to avoid name clashes. Applications reading this file must ignore unknown fields. Example: "DEBIAN_BTS="debbugs://bugs.debian.org/""

Container and sandbox runtime managers may make the host's identification data available to applications by providing the host's /etc/os-release (if available, otherwise /usr/lib/os-release as a fallback) as /run/host/os-release.
```
Not sure whether it is so generic as we think it is. As I am saying no hard feelings, just putting another option on the table.

Reply:
We should probably at least use `NAME`, `ID` or `ID_LIKE` as @Vogtinator suggested.

Reply:
agree.although keep in mind: `ID_LIKE` probably is not differentiable enough. `suse` vs `opensuse  suse`

Reply:
The `ID` should be `sle` for SUSE and `opensuse` for openSUSE. I already did the change.